lfucg-exactions/client/js/middlewares/apiMiddleware.js

```diff
@@ -34,13 +34,27 @@ export default function api({ getState, dispatch }) {
         if (shouldCallAPI && !shouldCallAPI()) {
             return next(action);
         }
-        const header = (global.Authorization && global.Authorization.length > 1) ? {
+        let stored_token;
+        try {
+            stored_token = localStorage.getItem('Token');
+        } catch (e) {
+            const index_token = document.cookie.indexOf('Token=');
+            if (index_token) {
+                const semicolon = index_token + 5 + document.cookie.substring(index_token).indexOf(';');
+                stored_token = document.cookie.substring(index_token, semicolon);
```

Comment:
is there a guaranteed ; at the end of Token=?

Reply:
Yes, when cookies are added a ';' is automatically added to separate various cookies.